### ISSUE TITLE
Update Headers for Find a Rep

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#70844]
We have some [early accessibility feedback](https://dsva.slack.com/archives/C05L6HSJLHM/p1701271694454669) for our Find a Rep product in staging.

> hi folks! took a peek at Find a rep in staging (YAY!) and found a couple of accessibility issues around headings. Should be > easy fixes:
> “Search for a representative” is H3, should be H2
> “How can each type of representative help me?” is H6, should be H3

### Tasks

- [ ] Update “Search for a representative” to H2
- [ ] Update “How can each type of representative help me?” to H3
